lib/cadence/testing/local_workflow_context.rb

```diff
@@ -61,7 +61,12 @@ def execute_activity(activity_class, *input, **args)
         )
         context = LocalActivityContext.new(metadata)
 
+      begin
         result = activity_class.execute_in_context(context, input)
+      rescue StandardError, ScriptError => error
+        future.fail(error.class.name, error.message)
+        return
```

Comment:
This needs to return the future back otherwise it won't allow testing failure cases